libtiledbsoma/src/utils/common.h

```diff
@@ -39,6 +39,10 @@
 
 namespace tiledbsoma {
 
+using MetadataValue =
+    std::tuple<std::string, tiledb_datatype_t, uint32_t, const void*>;
+enum MetadataInfo { key = 0, dtype, num, value };
```

Comment:
Given that `MetaDataInfo` is only used in test, wouldn't it make sense to move it there?

Reply:
I am planning on using this in code outside of the unit tests soon.